modules/ROOT/pages/7.0-release-notes.adoc

```diff
@@ -283,6 +282,17 @@ To address this, the summary element has been included in the list of non-empty
 
 With this adjustment, the summary element will no longer be deleted under the previously mentioned conditions.
 
+=== `TableOperations.opPasteColsAfter` & `TableOperations.opPasteRowsAfter` now place the cursor in the newly pasted row/column to be consistent with the other pasting operations.
+// #TINY-6460
+
+Previously, the cursor position in `opPasteRowsAfter` and `opPasteColsAfter` was calculated incorrectly after pasting a new row or column.
+
+As a consequence, after pasting, the cursor remained in the same position instead of moving to the newly pasted row or column.
+
+{productname} 7.0 addresses this, fixing the cursor calculation in `opPasteRowsAfter` and `opPasteColsAfter` to use the `index` variable.
+
+Now, when using `opPasteRowsAfter` and `opPasteColsAfter`, the cursor is correctly positioned in the newly pasted row or column as expected.
```

Comment:
Looking at https://github.com/tinymce/tinymce/pull/9244/files, a changelog entry was added for Snooker (our internal table library) but one wasn't added for the TinyMCE changelog. An entry needs to be added for the TinyMCE changelog as well

Potential suggestion:
```suggestion
=== The `mceTablePasteRowAfter` command, `mceTablePasteColAfter` command, `"Paste row after"` menu item and `"Paste column after"` menu item now place the cursor in the newly pasted row/column to be consistent with the other pasting operations.
// #TINY-6460

Previously, the cursor position was incorrectly calculated after pasting a new row or column when using the `mceTablePasteRowAfter` command, `mceTablePasteColAfter` command, or `"Paste row after"` and `"Paste column after"` UI menu items.

As a consequence, after pasting, the cursor remained in the same position instead of moving to the newly pasted row or column.

{productname} 7.0 addresses this, fixing the cursor calculation.

Now, when using the `mceTablePasteRowAfter` command, `mceTablePasteColAfter` command, or `"Paste row after"` and `"Paste column after"` UI menu items, the cursor is correctly positioned in the newly pasted row or column as expected.
```